src/Stratis.Bitcoin.Features.Notifications.Tests/BlockNotificationTest.cs

```diff
@@ -17,6 +17,8 @@ namespace Stratis.Bitcoin.Features.Notifications.Tests
 {
     public class BlockNotificationTest : LogsTestBase
     {
+        private const string SkipReason = "Revisit these tests when BlockNotification is fixed.";
```

Comment:
Create an issue for this so that it doesn't get lost when everyone is happy happy that we have a green build

Reply:
I created one earlier but didn't link it, thanks for spotting this!